src/KubeOps.Abstractions/Finalizer/IEventFinalizerAttacherFactory.cs

```diff
@@ -0,0 +1,22 @@
+using k8s;
+using k8s.Models;
+
+namespace KubeOps.Abstractions.Finalizer;
+
+/// <summary>
+/// Represents a type used to create <see cref="EntityFinalizerAttacher{TImplementation,TEntity}"/> for controllers.
+/// </summary>
+public interface IEventFinalizerAttacherFactory : IDisposable
```

Comment:
Same here, this factory basically just forwards 1 param. I don't see the benefit of having a factory for a delegate of this type.